### ISSUE TITLE
apk-update using HTTP by default

### ISSUE THREAD

**Issue by rvillane** [alpinelinux/docker-alpine#139]
On Dockerfile we have instructions to update openSSL package and by doing that we realized the APK registry is using HTTP by default.  Given security concerns, shouldn't it default to HTTPS?

```
FROM node:12-alpine

RUN mkdir /dist
WORKDIR /dist

COPY .npm* package*.json app.js *.json /dist/
COPY lib /dist/lib/
RUN apk update && apk add --upgrade openssl
```

here is the log we get as we have egress rules that prevent HTTP traffic.

```
fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz
[91mERROR: http://dl-cdn.alpinelinux.org/alpine/v3.11/main: network error (check Internet connection and firewall)
WARNING: Ignoring APKINDEX.70f61090.tar.gz: No such file or directory
[0mfetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz
[91mERROR: http://dl-cdn.alpinelinux.org/alpine/v3.11/community: network error (check Internet connection and firewall)
WARNING: Ignoring APKINDEX.ca2fea5b.tar.gz: No such file or directory
[0m2 errors; 16 distinct packages available
The command '/bin/sh -c apk update && apk add --upgrade openssl' returned a non-zero code: 2

```

any advice ?

**Comment by rvillane** [comment]
seems to be tightly coupled to issue: https://github.com/gliderlabs/docker-alpine/issues/184

**Comment by psmolkin** [comment]
Looks like alpine:3.13 changed it

**Comment by ncopa** [comment]
this was fixed in alpine:3.13.
